packages/Owl/Owl.0.1.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Liang Wang (ryanrhymes@gmail.com)"
+authors: [ "Liang Wang (ryanrhymes@gmail.com)" ]
+license: "MIT"
+homepage: "https://github.com/ryanrhymes/owl"
+dev-repo: "https://github.com/ryanrhymes/owl.git"
+bug-reports: "https://github.com/ryanrhymes/owl/issues"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocaml" "%{etc}%/Owl/_oasis_remove_.ml" "%{etc}%/Owl"]
+]
+build-test: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "ctypes"
+  "dolog"
+  "gsl"
+  "oasis" {build & >= "0.4"}
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
+  "plplot"
```

Comment:
while I agree the `conf-gsl` dependency should be explicit, I believe this won't solve the underlying issue -- which is a gsl version in your favourite Travis machine which is slightly too old to have sparse matrix support... nothing easily fixable (but e.g. the MacOSX Travis finished successfully).. I suspect most owl users will have more modern systems than whatever Travis is ;)